### DISCUSSION TITLE
emscripten support

### DISCUSSION THREAD

**Discussion by andreamancuso** [NVIDIA/stdexec#1344] [General]
Hi,

I just wanted to ask whether emscripten is supported and if not, whether you plan to add support for it as well as an example.

Thanks!

**Comment by ericniebler** [comment]
I have never tried building with emscripten, and I have no work planned. It will only be a thing if someone makes a contribution and commits[*] to maintaining it.

[*]: as much as anybody in open source can commit to these things

**Reply by andreamancuso** [reply]
Fair enough. Thanks for the prompt reply and for being upfront about it.

If I do end up using stdexec in my project I'll be sure to include relevant examples.

Best,
Andrea

**Comment by villevoutilainen** [comment]
I have this example
https://git.qt.io/vivoutil/libunifex-with-qt/-/blob/main/thread_example/CMakeLists.txt?ref_type=heads
built with the necessary support for Qt on WASM, and I have run it in a browser. In other words, based on my explorations, emscripten just works out of the box.

**Reply by andreamancuso** [reply]
Wonderful, thank you very much!
